### PR TITLE
add recipe for eomaps

### PR DESCRIPTION
**Pull request by raphaelquast**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by raphaelquast** [comment]
@conda-forge/core  I'm sorry but I have no clue why the linter is telling me that he couldn't find any recipes...

**Comment by raphaelquast** [comment]
@conda-forge-admin, please ping team 
I'm sorry but I have no clue why the linter is telling my that he couldn't find any recipes...

**Comment by raphaelquast** [comment]
@conda-forge-admin, please ping team

I finally managed to get this is ready for review!

File: recipes/eomaps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  requires:
+    - pip
+
+about:
+  home: https://github.com/raphaelquast/EOmaps
+  summary: A library to create interactive maps of geographical datasets.
+  license: GPL-2.0-or-later
+  license_file: LICENSE
+
+  # The remaining entries in this section are optional, but recommended.
```
Comment (chrisburr) [inline recipes/eomaps/meta.yaml:53]:
```suggestion
```

**Comment by raphaelquast** [comment]
@chrisburr wow that was a fast response! thanks a lot!!!

can you answer a small final question to make sure I understand how to maintain this in the future?

I get from the docs that you have a bot that automatically creates pull-requests if I release a new version on PIP right?
So all I have to do once a new version is released is to checkout the automatically created branch, fix whatever's not right, submit a new PR and wait for it to get merged again?

**Comment by sugatoray** [comment]
> I get from the docs that you have a bot that automatically creates pull-requests if I release a new version on PIP right? So all I have to do once a new version is released is to checkout the automatically created branch, fix whatever's not right, submit a new PR and wait for it to get merged again?

@raphaelquast Yes. Pretty much.

However, if you don't want to wait until the bot creates the PR, after the PyPI version is released, all you have to do is fork the feedstock (`conda-forge/eomaps-feedstock`) and create a new branch (say `update_v2.1.2`) and change the following:

- [x] change the version
- [x] no change to build number since new version update
- [x] change the sha256 (either use `openssl` or directly grab it from PyPI)
- [x] update any new dependency and drop any stale-dependency
- [x] update dependency versions under `requirements`

The raise a PR to merge it back to the feedstock. Follow the instructions you see while raising the PR from GitHub over a browser.

**Comment by raphaelquast** [comment]
@sugatoray 
aaah, OK that makes sense... thanks for the detailed explanation!

**Pull request merged**